### ISSUE TITLE
source:convert not creating document folder metadata

### ISSUE THREAD

**Issue by johnzhao9** [forcedotcom/cli#609]
### Summary

_Short summary of what is going on or to provide context_.
`source:convert` no longer adds Document Folder's .xml metadata (can't deploy via MDAPI as a result)
Works in `sfdx-cli/7.63.0 win32-x64 node-v12.18.2`. Works in the last version as well `sfdx plugins:install salesforcedx@49.9.2`
Fails in the latest `sfdx-cli/7.74.1-32db2396ed linux-x64 node-v12.18.3`

### Steps To Reproduce:

**Repository to reproduce:** [sfdx-v7-63-doc-xml-issue](https://github.com/johnzhao9/sfdx-v7-63-doc-xml-issue.git)

1.  Clone the repo above and `bash ./run`
1.  `ls out/documents/`

### Expected result
expected `TestFolder-meta.xml` to exist so it can be deployed via MDAPI

### Actual result
`TestFolder-meta.xml` does not exist; the folder cannot be deployed

### Additional information
This is used in our CI for a lot of things, copying over the `.xml` by hand is not a viable workaround.

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
`sfdx-cli/7.74.1-32db2396ed linux-x64 node-v12.18.3`

**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
```
root@8def2491d2dc:/sfdx-v7-63-doc-xml-issue# sfdx plugins --core
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.0 (core)
@oclif/plugin-update 1.3.10 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.4.3 (core)
alias 1.1.1 (core)
analytics 1.12.1 (core)
config 1.1.8 (core)
generator 1.1.3 (core)
salesforcedx 49.10.0 (core)
├─ salesforce-alm 49.11.0 (core)
├─ templates 49.4.4 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ custom-metadata 1.0.10 (core)
└─ apex 0.0.9 (core)
sfdx-cli 7.74.1 (core)
```

**OS and version**:
```
root@8def2491d2dc:/sfdx-v7-63-doc-xml-issue# uname -a
Linux 8def2491d2dc 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by shetzel** [comment]
The fix for this issue and #612 will be in the next patch release.

**Comment by johnzhao9** [comment]
Awesome thank you, will check it out when the patch comes out!

**Comment by shetzel** [comment]
This is fixed in: salesforcedx v50.5.0 (November 12, 2020) - CLI 7.80.0
